src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newObj = {};
+
+  for (const key in robot) {
+    if (newObj[robot[key]]) {
+      return null;
+    }
+    newObj[robot[key]] = key;
```

Comment:
A blank line between a block and the next command

```suggestion
    }

    newObj[robot[key]] = key;
```